rpc/tls.go

```diff
@@ -96,12 +96,29 @@ func LoadTLSConfig(certPEM, keyPEM, caPEM []byte) (*TLSConfig, error) {
 	return &TLSConfig{
 		config: &tls.Config{
 			Certificates: []tls.Certificate{cert},
-			ClientAuth:   tls.RequireAndVerifyClientCert,
-			RootCAs:      certPool,
-			ClientCAs:    certPool,
-
-			// TODO(jqmp): Set CipherSuites?
-			// TODO(jqmp): Set MinVersion?
+			// TODO(marc): clients are bad about this. We should switch to
+			// tls.RequireAndVerifyClientCert once client certs are properly set.
+			ClientAuth: tls.VerifyClientCertIfGiven,
+			RootCAs:    certPool,
+			ClientCAs:  certPool,
+
+			// Ciphers from http://golang.org/src/crypto/tls/cipher_suites.go
+			// but with RC4 and 3DES removed.
+			CipherSuites: []uint16{
+				tls.TLS_RSA_WITH_AES_128_CBC_SHA,
```

Comment:
Order matters here; we should follow the order at  https://github.com/golang/go/blob/357a013060686aa90432dba93faa01196452eabe/src/crypto/tls/cipher_suites.go#L74. 

It's also reasonable IMHO to just use go's defaults here. Go 1.4 does have rc4 by default but it's being removed from the list for 1.5 (it's not obvious but that's what the `suiteDefaultOff` in the above link means)

We should probably also set `PreferServerCipherSuites: true` so we use the server's order instead of the client's.